platform_versioned_docs/version-24.1/compute-envs/azure-batch.mdx

```diff
@@ -55,56 +59,56 @@ After creating a resource group, set up an [Azure storage account][az-learn-stor
     If you haven't created a resource group, you can do so now.
     :::
 
-2. Enter a name for the storage account (e.g., _towerrgstorage_).
-3. Choose the preferred region (same as the Batch account).
-4. The platform supports any performance or redundancy settings — select the most appropriate settings for your use case.
-5. Select **Next: Advanced**.
-6. Enable _storage account key access_.
-7. Select **Next: Networking**.
+1. Enter a name for the storage account (e.g., _towerrgstorage_).
+1. Choose the preferred region (same as the Batch account).
+1. The platform supports any performance or redundancy settings — select the most appropriate settings for your use case.
+1. Select **Next: Advanced**.
+1. Enable _storage account key access_.
+1. Select **Next: Networking**.
    - Enable public access from all networks. You can enable public access from selected virtual networks and IP addresses, but you will be unable to use Forge to create compute resources. Disabling public access is not supported.
-8. Select **Data protection**.
+1. Select **Data protection**.
     - Configure appropriate settings. All settings are supported by the platform.
-9. Select **Encryption**.
+1. Select **Encryption**.
     - Only Microsoft-managed keys (MMK) are supported.
-10. In **tags**, add any required tags for the storage account.
-11. Select **Review and Create**.
-12. Select **Create** to create the Azure Storage account.
-13. You will need at least one blob storage container to act as a working directory for Nextflow.
-14. Go to your new storage account and select **+ Container** to create a new Blob storage container. A new container dialogue will open. Enter a suitable name, e.g., _towerrgstorage-container_.
-15. Go to the **Access Keys** section of your new storage account (_towerrgstorage_ in this example).
-16. Store the access keys for your Azure Storage account, to be used when you create a Seqera compute environment.
+1. In **tags**, add any required tags for the storage account.
+1. Select **Review and Create**.
+1. Select **Create** to create the Azure Storage account.
+1. You will need at least one blob storage container to act as a working directory for Nextflow.
+1. Go to your new storage account and select **+ Container** to create a new Blob storage container. A new container dialogue will open. Enter a suitable name, e.g., _towerrgstorage-container_.
+1. Go to the **Access Keys** section of your new storage account (_towerrgstorage_ in this example).
+1. Store the access keys for your Azure Storage account, to be used when you create a Seqera compute environment.
 
 :::caution
 Blob container storage credentials are associated with the Batch pool configuration. Avoid changing these credentials in your Seqera instance after you have created the compute environment.
 :::
 
-## Batch account
+### Batch account
```

Comment:
both this header and the next "Create a batch account" are at the same level. Is it on purpose? 
Or should the second header be removed?